sdcm/utils.py

```diff
@@ -16,6 +16,7 @@
 import time
 import datetime
 import requests
+import subprocess
```

Comment:
why adding subprocess here ?

Reply:
used to download files from buckets. because using boto3 api return 403 code upon downloading.
rewrite using requests